dev-python/argon2-cffi-bindings/argon2-cffi-bindings-21.2.0.ebuild

```diff
@@ -0,0 +1,33 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_{8..10} pypy3 )
+inherit distutils-r1
+
+DESCRIPTION="Low-level CFFI bindings for the Argon2 password hashing library"
+HOMEPAGE="https://github.com/hynek/argon2-cffi-bindings"
+SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="cpu_flags_x86_sse2"
+
+DEPEND="
+	app-crypt/argon2:=
+	virtual/python-cffi[${PYTHON_USEDEP}]
```

Comment:
This one's a BDEP+RDEP, I think.

Reply:
I first looked at some other Python packages. They seem to add the virtual to `DEPEND`. I then checked the python-cffi docs, but couldn't find anything specific. Lastly, I has a look at the resulting .so file and it only links against argon2, but not against libffi:
```
$ ldd /usr/lib/python3.9/site-packages/_argon2_cffi_bindings/_ffi.abi3.so
        linux-vdso.so.1 (0x00007ffd4abec000)
        libargon2.so.1 => /usr/lib64/libargon2.so.1 (0x00007efc22e8e000)
        libc.so.6 => /lib64/libc.so.6 (0x00007efc22ccf000)
        libpthread.so.0 => /lib64/libpthread.so.0 (0x00007efc22caf000)
        /lib64/ld-linux-x86-64.so.2 (0x00007efc22ec8000)
```
I suppose `BDEPEND` is the correct dependency for `virtual/python-cffi` then?

Reply:
Yes.

Reply:
I'm not going to insist on this but wouldn't it be simpler to declare `DEPEND` and `BDEPEND` first, then do `RDEPEND="${DEPEND} ${BDEPEND}"`?

Reply:
Sure, I don't have a strong opinion on this.

I adjusted the dependencies `argon2-cffi-bindings-21.2.0.ebuild` accordingly.